scripts/script_utils.py

```diff
@@ -113,6 +117,7 @@ def get_cache_service(manager):
         return manager.InMemoryCacheService()
 
 
+# flake8: noqa: C901
```

Comment:
I added this decorator to ignore Flak8 to get my code commited.  Currently complexity is 13 for get_rules and it has to be below 10 for flake8 to not block commits.  I was interested in your thoughts @gerrycampion on how I could get it down under 10

Reply:
Yes, this method is pretty complex and I think it also needs a little work because the args.rules filter should work regardless of the source.
I would reduce the complexity by creating separate methods for at least:
- getting rules from pickle cache
- getting rules from local files